src/tests/test_convert_gmp_to_decimal.c

```diff
@@ -0,0 +1,35 @@
+#include "s21_decimal_test.h"
```

Comment:
Thank you very much for writing unit tests for this code. Really appreciate it! 😇